examples/creatives/banner_nonlinear.js

```diff
@@ -0,0 +1,51 @@
+class BannerNonLinear extends BaseSimidCreative{
+    constructor() {
+        super();
+        this.addButtonClickActions_();
+    }
+
+    /** @override */
+    onInit(eventData) {
+        super.onInit(eventData);
+        this.dynamicResize_();
+    }
+
+    /**
+     * Adds actions to different buttons available on the overlay.
+     * @private
+     */
+    addButtonClickActions_() {
+        this.sendMessageOnButtonClick_("close_ad", CreativeMessage.REQUEST_STOP);
+        this.sendMessageOnButtonClick_("ad_text", CreativeMessage.REQUEST_EXPAND);
+        this.sendMessageOnButtonClick_("minimize_ad", CreativeMessage.REQUEST_COLLAPSE);
+    }
+
+    /**
+     * Listens for a click event on a button
+     * @param {String} elementName The name of the element.
+     * @param {String} message The message to send to the player.
+     * @private
+     */
+    sendMessageOnButtonClick_(elementName, message) {
+        const sendMessageFunction = () => {this.simidProtocol.sendMessage(message);}
+        document.getElementById(elementName).addEventListener(
+            'click', sendMessageFunction);
+    }
+    /**
+     * Repositions the banner ad according to the dimensions of the video player
+     * by calculating desired dimensions and sending a resize request to creative.
+     * @private
+     */
+    dynamicResize_() {
+        let creativeDimensions = {};
+        creativeDimensions.x = this.environmentData.videoDimensions.width * .15;
```

Comment:
Are these numbers (.15 and .7) arbitrary? Maybe make them to consts

Reply:
good call, updated these to be constants